dockstore-common/src/main/scala/io/dockstore/common/WdlBridge.scala

```diff
@@ -105,15 +105,19 @@ class WdlBridge {
     bundle.allCallables.foreach(callable => {
       callable._2 match {
         case w: WorkflowDefinition => {
-          val before = callable._2.asInstanceOf[WorkflowDefinition].meta.mapValues(_.asInstanceOf[MetaValueElementString].value)
-          val metadata = JavaConverters.mapAsJavaMap(before)
+          val workflow_instance = callable._2.asInstanceOf[WorkflowDefinition]
+          val workflow_metadata = workflow_instance.meta
+          val converted_workflow_map = workflow_metadata.map{ case (k, v) => (k, v.asInstanceOf[MetaValueElementString].value)}
+          val metadata = JavaConverters.mapAsJavaMap(converted_workflow_map)
           if (!metadata.isEmpty) {
             metadataList.add(metadata)
           }
         }
         case c: CallableTaskDefinition => {
-          val before = callable._2.asInstanceOf[CallableTaskDefinition].meta.mapValues(_.asInstanceOf[MetaValueElementString].value)
-          val metadata = JavaConverters.mapAsJavaMap(before)
+          val task_instance = callable._2.asInstanceOf[CallableTaskDefinition]
```

Comment:
Same question, was this changed for readability? Trying to understand what I should be looking for.

Reply:
Yes see above.